index.html

```diff
@@ -20,223 +20,303 @@
 </head>
 
 <body>
-    <header>
-        <a class="logo" href="#">My Logo</a>
-        <nav class="Union"><a href="#"><img src="images/Union.png" alt="menu"></a></nav>
-        <nav class="menu1">
-            <ul>
-                <li><a href="#portfolio">Portfolio</a> </li>
-                <li><a href="#about">About</a></li>
-                <li><a href="#contact">Contact</a></li>
+
+    <header id="header">
+        <div class="my-logo-container">
+            <a href="/">
+                <img class="my-logo" src="images/MyLogo.png" alt="Portfolio logo">
+            </a>
+        </div>
+        <nav>
+            <div class="menu-container">
+                <img src="images/Union.png" alt="menu" class="menu">
+            </div>
+            <ul class="nav-links">
+                <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+                <li class="nav-link"><a href="#about">About</a></li>
+                <li class="nav-link"><a href="#contact">Contact</a></li>
```

Comment:
All in your website looks good, but I encourage you to add more styles in order to have a website more similar to the Figma design you selected.

- [ ] [OPTIONAL] Add images, vectors missing.

Your design | Figma design

![image](https://user-images.githubusercontent.com/60164239/131533757-e0776784-5d72-41e6-993d-16821c841048.png)

Reply:
Thanks I will look into that